packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,107 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+- [Library](#library)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop          | Type                                                                     | Default     | Description                                                               | Required |
+| ------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------- | -------- |
+| content       | `string`                                                                 | `undefined` | Content of the tooltip                                                    | ✅       |
+| placement     | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip                                                      | ✅       |
```

Comment:
Do we've usecase for top-left, left-bottom, etc.?

Reply:
Didn't understand 🤔 top-left == top-start, left-bottom == bottom-start can you visually show the difference?

Reply:
So basically if the tooltip arrow was towards left or right:

```
------
|         |
|         |
|         |> (do we need to control this placement?)
------
```

Reply:
Not seeing this on the design, I think in this case, if the tooltip's height is longer and placement=left the arrow will just stay on the middle. 

<img width="399" alt="image" src="https://github.com/razorpay/blade/assets/35374649/866722be-8dad-44e5-bcc6-541650e19c5f">